src/test/java/ru/training/at/hw2/ex1/TestEx1.java

```diff
@@ -0,0 +1,140 @@
+
+        Assert.assertEquals(home, "HOME");
+        Assert.assertEquals(contact, "CONTACT FORM");
+        Assert.assertEquals(service, "SERVICE");
+        Assert.assertEquals(metals, "METALS & COLORS");
+    }
+
+    //6.Assert that there are 4 images on the Index Page and they are displayed
+    @Test(priority = 5)
+    public void testImageEquals() {
+        List<WebElement> listImages = webDriver.findElements(By.cssSelector("div.benefit-icon"));
+        Assert.assertEquals(listImages.size(), TestData.NUMBER_OF_IMAGES);
+        for (WebElement webElement: listImages) {
+            Assert.assertTrue(webElement.isDisplayed());
+        }
+    }
+    //7.Assert that there are 4 texts on the Index Page under icons and they have proper text
+    @Test(priority = 6)
+    public void testTextUnderIcons() {
+        List<WebElement> webElementList = webDriver.findElements(By.cssSelector("span.benefit-txt"));
+        List<String> stringList = new ArrayList<>();
+        for (WebElement webElement: webElementList) {
+            stringList.add(webElement.getText());
+        }
+        Assert.assertEquals(stringList.size(), TestData.TEXT_UNDER_ICONS.size());
+    }
+    //8.Assert that there is the iframe with “Frame Button” exist
+    @Test(priority = 7)
+    public void testIframeButton() {
+        WebElement iFrameButton = webDriver.findElement(By.tagName("iframe"));
+        Assert.assertTrue(iFrameButton.isDisplayed());
+    }
+    //9.Switch to the iframe and check that there is “Frame Button” in the iframe
+    @Test(priority = 8)
+    public void testFrameButtonInIframe() {
+        webDriver.switchTo().frame("frame");
+        webElement = webDriver.findElement(By.id("frame-button"));
+        Assert.assertTrue(webElement.isDisplayed());
+    }
+    //10.Switch to original window back
+    @Test(priority = 9)
+    public void checkOriginalWindow() {
+        webDriver.switchTo().defaultContent();
+        Assert.assertEquals(webDriver.getWindowHandle(), originalHandle);
+    }
+    //11.Assert that there are 5 items in the Left Section are displayed and they have proper text
+    @Test(priority = 10)
+    public void testLeftSectionItems(){
+        List <WebElement> leftSectionItems = webDriver.findElements(By.cssSelector("ul.sidebar-menu.left > li > a > span"));
+        for (WebElement webElement: leftSectionItems) {
+            Assert.assertTrue(webElement.isDisplayed());
+        }
+        Assert.assertEquals(leftSectionItems.get(0).getText(), TestData.LEFT_SECTION_ITEMS_TEXTS.get(0));
+        Assert.assertEquals(leftSectionItems.get(1).getText(), TestData.LEFT_SECTION_ITEMS_TEXTS.get(1));
+        Assert.assertEquals(leftSectionItems.get(2).getText(), TestData.LEFT_SECTION_ITEMS_TEXTS.get(2));
+        Assert.assertEquals(leftSectionItems.get(3).getText(), TestData.LEFT_SECTION_ITEMS_TEXTS.get(3));
+        Assert.assertEquals(leftSectionItems.get(4).getText(), TestData.LEFT_SECTION_ITEMS_TEXTS.get(4));
+    }
```

Comment:
Where is in the requirements specifying that task should be implemented as set of tests?